### ISSUE TITLE
Scaling Mode change during encode crashes VidCoder

### ISSUE THREAD

**Issue by surlias** [RandomEngy/VidCoder#137]
I've discovered that if you try to change the "Scaling Mode" on a preset while an encode is in progress, VidCoder will crash. To recreate the issue I changed Scaling Mode from "Downscale only" to "Scale up/down to target", which instantly resulted in a crash and the following error details:

```
System.Exception: An OnError occurred on an object (usually ObservableAsPropertyHelper) that would break a binding or command. To prevent this, Subscribe to the ThrownExceptions property of your objects ---> System.InvalidOperationException: No width or height limits were found.
   at VidCoderCommon.Model.JsonEncodeFactory.GetOutputSize(VCProfile profile, SourceTitle title) in D:\docs\projects\VidCoder\VidCoderCommon\Model\JsonEncodeFactory.cs:line 1204
   at VidCoder.Services.OutputSizeService.Refresh() in D:\docs\projects\VidCoder\VidCoder\Services\OutputSizeService.cs:line 42
   at System.Reactive.AnonymousSafeObserver`1.OnNext(T value)
   at System.Reactive.Linq.ObservableImpl.Select`2._.OnNext(TSource value)
   at System.Reactive.Linq.ObservableImpl.DistinctUntilChanged`2._.OnNext(TSource value)
   at System.Reactive.Linq.ObservableImpl.Select`2._.OnNext(TSource value)
   at System.Reactive.Linq.ObservableImpl.Where`1._.OnNext(TSource value)
   at System.Reactive.Linq.ObservableImpl.Switch`1._.Iter.OnNext(TSource value)
   at System.Reactive.Linq.ObservableImpl.Select`2._.OnNext(TSource value)
   at System.Reactive.Linq.ObservableImpl.Select`2._.OnNext(TSource value)
   at System.Reactive.Linq.ObservableImpl.Where`1._.OnNext(TSource value)
   at System.Reactive.Linq.ObservableImpl.Cast`2._.OnNext(TSource value)
   at System.Reactive.Observer`1.OnNext(T value)
   at System.Reactive.Linq.ObservableImpl.SelectMany`2.NoSelectorImpl.OnNext(TSource value)
   at System.Reactive.Linq.ObservableImpl.Buffer`2.Beta.Omega.OnNext(TBufferClosing value)
   at System.Reactive.Linq.ObservableImpl.Merge`1._.Iter.OnNext(TSource value)
   at System.Reactive.Linq.ObservableImpl.Select`2._.OnNext(TSource value)
   at System.Reactive.Linq.ObservableImpl.Where`1._.OnNext(TSource value)
   at System.Reactive.Observer`1.OnNext(T value)
   at ReactiveUI.IReactiveObjectExtensions.ExtensionState`1.notifyObservable[T](TSender rxObj, T item, ISubject`1 subject)
   --- End of inner exception stack trace ---
   at ReactiveUI.RxApp.<>c__DisplayClass5.<.cctor>b__2()
   at System.Reactive.Concurrency.Scheduler.Invoke(IScheduler scheduler, Action action)
   at System.Reactive.Concurrency.DispatcherScheduler.<>c__DisplayClass1`1.<Schedule>b__0()
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

**Comment by RandomEngy** [comment]
Thanks for the report. Will have a fix in the next Beta.

**Comment by RandomEngy** [comment]
Fixed in 2.36 Beta.
